rmw_connext_cpp/CMakeLists.txt

```diff
@@ -93,9 +87,12 @@ set(generated_files
 )
 add_custom_command(
   OUTPUT ${generated_files}
-  COMMAND ${PYTHON_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/bin/call-idl-pp.py" --idl-pp ${_idl_pp} --idl-file "connext_static_serialized_data.idl" -d ${generated_directory}
+  # We *explicitly* do not use Connext_DDSGEN_SERVER here.  That's so that we don't
+  # accidentally connect to another version of the server that might still be running
+  # from previous runs in rosidl_typesupport_connext_cpp.
```

Comment:
Please add to this comment that the other instance is invoked with different arguments and they seem to leak into an invocation from here.

Reply:
Done in ec5638a